File: Intermediate/Java/problem_24.java
/* 24. Display the pattern below
      A
      A B 
      A B C
      A B C D
*/

public class problem_24 {
    public static void main(String[] args) {
        for(int i=1;i<5;i++){
            char j ='A';
            for(int k=0;k<i;k++){
                System.out.print(j+" ");
                j++;
            }
            System.out.println();
        }
    }
}

File: Intermediate/C/problem_24.c
/*  display the pattern below
A
A B
A B C 
A B C D*/

#include<stdio.h>
void main()
{int a,i,j;
char b;
 printf("Enter a number:");
 scanf("%d",&a);

 for(i=0;i<a;i++)
 {  b=65;
   for(j=0;j<=i;j++)
   {printf("%c ",b);
   b++;
   }
   printf("\n");
 }
}
